### DISCUSSION TITLE
**PLEASE** add support for Elantouch trackpads

### DISCUSSION THREAD

**Discussion by NDL803** [elementary/triage#512] [Ideas]
Many laptops use Elantouch trackpads; Linux distributions don't support these devices, forcing users to use a mouse, or to use Windows.

Please include drivers for Elantouch trackpads in Elementary 8.  Many Linux users would appreciate it.

Thank you :)

**Comment by teamcons** [comment]
If they dont support it out of the box, then it is because there is no driver for it in the kernel.

that would be something to discuss with the linux kernel dev team, i suppose ?

**Comment by danirabbit** [comment]
Hey thanks for your feedback. I'm going to go ahead and close this discussion post since as the other commenter pointed out driver development happens upstream in the Linux kernel. elementary doesn't develop drivers so there's no action for us to take here. Thanks for understanding!
